php/CacheBuster/CacheBuster.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+
+namespace Base\CacheBuster;
+
+
+class CacheBuster {
+
+    /**
+     * @var string
+     */
+    private $version = '';
+
+    const ENVIRONMENT_VARIABLE_NAME = 'BASE_CACHE_VERSION';
+
+    public function __construct() {
+
+        $environmentVersion = getenv(self::ENVIRONMENT_VARIABLE_NAME);
+        if($environmentVersion && $environmentVersion !== '') {
+            $this->version = $environmentVersion;
+            return;
+        }
+
+        if (defined('WP_ENV') && WP_ENV === 'development') {
+            // if in development mode then make each request get the file from the server
+            $this->version = (new TimestampVersionIdentifier())->getVersion();
+            return;
+        }
```

Comment:
Yeah I think I was just trying to give it a default way of working without the user of it having to instantiate a load of objects - ie. give it a simple interface